CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 Documentation for rocThrust available at
 [https://rocm.docs.amd.com/projects/rocThrust/en/latest/](https://rocm.docs.amd.com/projects/rocThrust/en/latest/).
 
+## (Unreleased) rocThrust 3.x.x for ROCm 6.1
```

Comment:
@Naraenda All tests pass. Given that the only change for ROCm 6.1 (so far) is this minor bugfix, can you update the rocThrust version to be 3.0.1?  And also in https://github.com/ROCm/rocThrust/blob/develop/CMakeLists.txt#L125.  Then we can merge this in.  Thanks!